authors/apps/authentication/tests/data.py

```diff
@@ -152,3 +154,23 @@
 decode_error = 'Invalid authentication. Could not decode token'
 
 prefix_error = 'Bearer is expected as the prefix'
+
+# Test Data for Social Login
+
+wrong_fb_token = {
```

Comment:
@tomuhenry I don't think these token are required here.

Reply:
Most of that is dummy data... It would probably not matter what you input as a token. It just needs some kind of well-formated input. Fixing it now